examples/gnrc_border_router/README.md

```diff
@@ -26,6 +26,7 @@ For `native` the host-facing [`netdev_tap`](https://doc.riot-os.org/netdev__tap_
 is configured, providing connectivity via a TAP interface to the RIOT instance.
 On the node-facing side [`socket_zep`](https://doc.riot-os.org/group__drivers__socket__zep.html)
 is used to simulate a IEEE 802.15.4 network.
+Set `UPLINK=tap` to re-use an existing TAP interface instead of creating a new one.
```

Comment:
Why not use this to define the TAP that is supposed to be reused:

```suggestion
Set `UPLINK=tapX` to re-use an existing TAP interface `tapX` instead of creating a new one.
```

Reply:
(then one can also check if the interface supposed to be re-used actually exists as well)

Reply:
`native` border router *always* requires a tap interface, you can pass it via the `PORT` variable.

If this tap interface does not exist, it will fail to launch already.

I don't think having a second way to specify the tap interface makes sense.